src/components/Divider/index.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react'
```

Comment:
wdyt about calling this file `ArrowDivider`? If we need to use the Divider without the arrow we can directly use the one from SRC.

Reply:
FWIW I think it's more straight-forward to always use the same divider component, with an optional arrow prop.

Reply:
makes sense